### ISSUE TITLE
Issue with loading the datasets in readthedocs

### ISSUE THREAD

**Issue by rosecers** [cosmo-epfl/scikit-cosmo#76]
https://scikit-cosmo.readthedocs.io/en/latest/read-only-examples/PlotGFRE.html#

See attachment for current errors
[Global Feature Reconstruction Error (GFRE) and Distortion (GFRD) — scikit-COSMO 0.1.0 documentation.pdf](https://github.com/cosmo-epfl/scikit-cosmo/files/6067217/Global.Feature.Reconstruction.Error.GFRE.and.Distortion.GFRD.scikit-COSMO.0.1.0.documentation.pdf)

**Comment by agoscinski** [comment]
I am not sure where this comes from, the npz files are included in the installation, checked with
```bash
python setup.py bdist_egg | grep npz
```
Output
```
copying build/lib/skcosmo/datasets/data/degenerate_CH4_manifold.npz -> build/bdist.linux-x86_64/egg/skcosmo/datasets/data
copying build/lib/skcosmo/datasets/data/csd-1000r.npz -> build/bdist.linux-x86_64/egg/skcosmo/datasets/data
```
The test should also fail if this weren't true.
When I built the doc locally it also works (I made a new environment to test it).

I am not sure how the github page process exactly works, and where to look. Do you have any idea?

**Comment by Luthaf** [comment]
> I am not sure how the github page process exactly works, and where to look. 

The doc is not build on github pages but readthedocs.org. If you give me your username there I can give you access to their build log.

**Comment by agoscinski** [comment]
I think the error in the doc is because the necessary .npz files are in a packed python .egg archive so they cannot be accessed in as for a directory, I could change the type of installation using e.g. pip so no egg archive is used.
How can I change the building commands of readthedocs? (the ones in https://readthedocs.org/projects/scikit-cosmo/builds/13459127/)
I read about a `.readthedocs.yaml` in https://docs.readthedocs.io/en/stable/config-file/v2.html but couldn't find it so I guess we are using the default yml file?
One would need to add in the yml file method and pip https://docs.readthedocs.io/en/stable/config-file/v2.html#packages 
Something like
```
version: 2

python:
  version: 3.7
  install:
    - method: pip
      path: .
```
But I am really not sure about this, one could make a branch and activate it in the readthedocs to play around with this yml file, but I have not the rights to push directly branches to the skcosmo.

**Comment by rosecers** [comment]
Addressed by #96
